explorer/app_settings.py

```diff
@@ -17,7 +18,13 @@
     ('csv', 'explorer.exporters.CSVExporter'),
     ('excel', 'explorer.exporters.ExcelExporter'),
     ('json', 'explorer.exporters.JSONExporter'),
+
 ])
+if sys.version_info[0] < 3:
```

Comment:
Hello @chrisclark , I've added it in a try catch with the correct import - if python version > 2 or django-xhtml2pdf is not installed pdf exported will not be included.